gatsby_fp/src/components/fieldComponents/profService.js

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+
+export default function ProfService ({profServiceList}){
+  return(
+    
+    <>
+    <div className="sectiontitle">Professional Service</div>
+        <div>
+          <ul>
+          {profServiceList.map((element, index) => (
+            // <li key={index} dangerouslySetInnerHTML ={{__html: 
```

Comment:
Since we have the benefit of revision control, we can already jump back and forth between versions. Accordingly, we should avoid keeping code commented out.